### DISCUSSION TITLE
can't access to nextjs.org

### DISCUSSION THREAD

**Discussion by duccanhole** [vercel/next.js#42460] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

I try to access to nextjs.org page but it's unavailable.
![image](https://user-images.githubusercontent.com/62925811/199740677-f492a30f-56a9-4e08-a372-15792c0afb78.png)

### What browser are you using? (if relevant)

chrome, firefox

### Describe the Bug

can't access to nextjs.org

### Expected Behavior

the page work again

### Link to reproduction

nextjs.org

### To Reproduce

access to nextjs.org

**Comment by firatgh** [comment]
![Screenshot 2022-11-03 at 14 34 42](https://user-images.githubusercontent.com/96059813/199750134-600ead62-5893-4d39-ab7d-3a550e4ae070.png)

**Comment by duccanhole** [comment]
I just check window diagnosis, it show this
![image](https://user-images.githubusercontent.com/62925811/199753556-ca01fdd8-afb4-48ac-b696-2f4d2d4a6aaf.png)

**Comment by icyJoseph** [comment]
Is this happening on a "company" computer? Perhaps the domain is not allowed by your network admin? Or have you modified your DNS resolver?
